spidermon/contrib/validation/cerberus/validator.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import
+
+from cerberus import Validator as something
```

Comment:
You did right providing an alias for `cerberus.Validator` (as it conflicts with `spidermon.contrib.validation.validator.Validator`), but `something` is not a good alias as it has no useful meaning.

Some options would be:
`from cerberus import Validator as cerberus_validator`
or
`import cerberus` and use `cerberus.Validator` instead of `something`.